assets/src/edit-story/components/thumbnail/stories/index.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { action } from '@storybook/addon-actions';
+import { boolean, text } from '@storybook/addon-knobs';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { Tooltip, Icons } from '../../../../design-system';
+import { Thumbnail, THUMBNAIL_TYPES } from '..';
+import { THUMBNAIL_BG } from './demoThumbnails';
+
+export default {
+  title: 'Stories Editor/Components/Thumbnail',
+  component: Thumbnail,
+};
+const Container = styled.div`
+  background-color: #28292b;
+  padding: 30px;
+  display: flex;
+  gap: 20px;
+`;
+
+const StyledVideoOptimizationIcon = styled(Icons.GearWithGauge)`
```

Comment:
colors get wiped away for `currentColor` in `Icons` so reattaching the intended colors here.